resources/schema.json

```diff
@@ -73,6 +73,10 @@
                             "description": "Mutation score badge for your GitHub project."
                         }
                     }
+                },
+                "github": {
+                    "type": "string",
+                    "description": "GitHub Annotations for escaped Mutants in the added/modifies files"
```

Comment:
Is this relevant?

Reply:
yes, why not? 

> Now it's possible to use `--logger-github=AM` or
> 
> ```json
> {
>     "logs": {
>         "github": "A"
>     }
> }
> ```

---

hm, now I'm thinking how we will use `--logger-github` option. Without the value? So that if it's used withut `--git-diff-filter=A` - all the files will be mutated and too many annotations will be created. I don't say it's bad, but it's the fact.

Reply:
How about let them be? Say, one might have a small project with only minimal number of remaining mutations, and they want to fix them all. Having Infection report all and every will be a huge help.